apps/web/middleware.ts

```diff
@@ -76,15 +75,34 @@ const middleware: NextMiddleware = async (req) => {
   });
 };
 
+const routingForms = {
+  handle: (url: URL) => {
+    // Next.config.js Redirects don't handle Client Side navigations and we need that.
+    // So, we add the rewrite here instead.
+    if (url.pathname.startsWith("/routing-forms/")) {
+      url.pathname = url.pathname.replace(/^\/routing-forms\//, "/apps/routing-forms/");
```

Comment:
Made the replacement very specific using RegExp so that any other instance of `routing-forms `isn't replaced accidentally

Reply:
`/apps/routing_forms` worked before now it doesn't anymore. Why did we need that before and why don't we need it anymore?